src/components/pages/logInPage/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from './logInPage';
```

Comment:
I think it's fair to use just 'login' as the path here instead of 'logInPage' since it's already in the '/pages' directory. what do you think?